setup.py

```diff
@@ -340,29 +340,24 @@ def run_tests(self):
         sys.exit(main())
 
 
-class BinaryDistApp(PyInstallerDist):
-    description = 'create an application for OSX'
-    extra_args = [
-        '--icon=osx/plover.icns',
-        '--osx-bundle-identifier=org.openstenoproject.plover',
-    ]
+class BinaryDistApp(Command):
+    description = 'create an application bundle for Mac'
+    user_options = []
+    extra_args = []
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
 
     def run(self):
-        PyInstallerDist.run(self)
-        # Patch Info.plist.
-        plist_info_fname = 'dist/%s.app/Contents/Info.plist' % PACKAGE
-        tree = ElementTree.parse(plist_info_fname)
-        dictionary = tree.getroot().find('dict')
-        for name, value in (
-            # Enable retina display support.
-            ('key'   , 'NSPrincipalClass'),
-            ('string', 'NSApplication'   ),
-        ):
-            element = Element(name)
-            element.text = value
-            element.tail = '\n'
-            dictionary.append(element)
-        tree.write(plist_info_fname)
+        whl_cmd = self.get_finalized_command('bdist_wheel')
+        whl_cmd.run()
+        wheel_path = whl_cmd.get_archive_basename()
+        cmd = 'bash osx/make_app.sh %s %s' % (wheel_path, PACKAGE)
+        log.info('running %s', cmd)
+        subprocess.check_call(cmd.split())
```

Comment:
We had this discussion: https://github.com/openstenoproject/plover/pull/740#pullrequestreview-33657417 but I can still change it if you insist